tests/functional/api/kvs_api_test.c

```diff
@@ -183,31 +183,6 @@ MTF_DEFINE_UTEST(kvs_api_test, create_null_kvdb)
     ASSERT_EQ(EINVAL, hse_err_to_errno(err));
 }
 
-MTF_DEFINE_UTEST(kvs_api_test, create_max)
```

Comment:
Why was this test deleted?

Reply:
it's repeated in the limits test

Reply:
I don't think it makes sense to move it to a file called `cursor_txn_test.c`

Reply:
true, let me rename this file.